src/components/Button/Button.js

```diff
@@ -24,7 +24,7 @@ Btn.propTypes = {
   disabled: PropTypes.bool,
   children: PropTypes.node,
   onPress: PropTypes.func,
-  style: PropTypes.object,
+  styles: PropTypes.object,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop name has been changed from `style` to `styles`. This could potentially break any components that are using this Button component and passing a `style` prop. If this change is necessary, make sure to update all instances where the Button component is used to pass `styles` instead of `style`.
```diff
-  style: PropTypes.object,
+  styles: PropTypes.object,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->